tasks/copy_to_glacier/test/test_handler.py

```diff
@@ -54,15 +58,420 @@ def test_4_get_bucket(self):
             event = json.load(f)
             filename = path.basename(event.get('input', [])[0])
             files = event.get('config').get('collection').get('files', [])
-            bucket = get_bucket(filename, files)
+            bucket = get_bucket_name_for_filename(filename, files)
             self.assertEqual(bucket, 'protected')
 
-    def test_5_task(self):
+    def test_task_happy_path(self):
         """
-        Testing task
+        Basic path with buckets present.
         """
-        context = []
-        with open(self.event_path) as f:
-            event = json.load(f)
-            test_handler = task(event, context)
-            print(test_handler)
+        collection_name = uuid.uuid4().__str__()
```

Comment:
I think it would make sense to move this block of variable initialization into its own helper function. That should cut down on a decent amount of the duplicate code.

Reply:
Another option would be to just initialize one event object and pass a copy of that object to each test. If you don't have anything relying on the attributes being different for each test, that is. <- this can be done with the `setUp` and `tearDown` functions.

Reply:
https://github.com/nasa/cumulus-orca/blob/develop/tasks/copy_files_to_archive/test/test_copy_files_to_archive.py#L28 setup and teardown example

Reply:
Sadly, the event does have to be different. Both in terms of raw values and in terms of attributes being present or not present. I could have a helper function that gets passed a bunch of bools (set_property = false, set_another_property = false, set_a_third_property = false) but that might get a bit messy.
Also note that we need to know what these values are when writing the asserts, so we'd need to pass a bool into the helper function along with a value.

Reply:
I think I see where you're coming from. I was thinking about a helper function that just returns a dictionary of uuid4s so you'd be able to pick and choose which ones to use, but the trade-off is that you have to index into the dictionary for each variable reference. With the context you provided I think its fine to ignore my previous suggestions.